volumes/ecocounter/readme.md

```diff
@@ -1,17 +1,11 @@
 <!-- TOC -->
 
-- [Bicycle loop detectors](#bicycle-loop-detectors)
-  - [Installation types](#installation-types)
-  - [Ecocounter data](#ecocounter-data)
-    - [Flows - what we know](#flows---what-we-know)
-  - [Discontinuities](#discontinuities)
-  - [Using the Ecocounter API](#using-the-ecocounter-api)
-    - [Note](#note)
-  - [Historical data](#historical-data)
-  - [`ecocounter_pull` DAG](#ecocounter_pull-dag)
-    - [`check_partitions` TaskGroup](#check_partitions-taskgroup)
-    - [`data_checks` TaskGroup](#data_checks-taskgroup)
-  - [`ecocounter_check` DAG](#ecocounter_check-dag)
+    - [Discontinuities](#discontinuities)
```

Comment:
somehow formatted as code block

Reply:
Fixed! And added back a missing section of readme 👀